README.md

```diff
@@ -117,6 +117,30 @@ naive and ignore the hostname verification:
 
     naive+https://public:private@host:port/1
 
+### Proxying HTTP(S) connections
+If your application sends outbound requests through an HTTP proxy, you can use
+JVM networking properties to configure the proxy information.
+
+E.g.
+
+`java \
```

Comment:
This renders a bit strange here: https://github.com/jszwedko/raven-java/blob/add-proxy-info-to-readme/README.md#proxying-https-connections

It is probably easier to just use triple backticks for a block code sample here.